src/screens/SignInScreen.tsx

```diff
@@ -1,14 +1,20 @@
-import { useContext } from "react";
-import { StyleSheet, Image } from "react-native";
+import { useContext, useEffect, useState } from "react";
+import { StyleSheet, Image, View } from "react-native";
 
-import { View } from "@ant-design/react-native";
+import * as AppleAuthentication from "expo-apple-authentication";
 
 import { StyledText } from "@components/StyledText";
 import { ThemeContext } from "@contexts/theme";
 import { GoogleSignInButton } from "@components/GoogleSignInButton";
+import { AppleSignInButton } from "@components/AppleSignInButton";
 
 export const SignInScreen = () => {
   const { theme } = useContext(ThemeContext);
+  const [isAppleLoginAvailable, setIsAppleLoginAvailable] = useState(false);
+
+  useEffect(() => {
+    AppleAuthentication.isAvailableAsync().then(setIsAppleLoginAvailable);
```

Comment:
This is such a weird thing. Sometimes it's not available? Why did they design it like this.

Reply:
Come to think of it can you test it on Android and check the layout? You will have to rebuild, but no rush since App Review team will only see iOS view

Reply:
Sure